### PR TITLE
Documentation(EJ2-66488):Added UG contents for newly added features in Schedule and Navigation components in ASP MVC/core

### PR DESCRIPTION
**Pull request by VenkateshwaranSaravanakumar**
Description:
1.Added UG contents for newly added feature on tab key navigation using tabIndex property in Tab and Toolbar components.
2.Added UG contents for newly added features hideNonWorkingDays property for displaying custom working days and customizing date range text using dateRangetemplate and also updated content for contextmenu.

Areas affected and ensured:
All possible cases

Test cases:
NA

Test bed sample location:
NA

Additional checklist
Did you run the automation against your fix? NA
Is there any API name change? No
Is there any existing behavior change of other features due to this code change? No
Does your new code introduce new warnings or binding errors? No
Does your code pass all FxCop and StyleCop rules? NA
Did you record this case in the unit test or UI test? No

### CODE REVIEW

File: ej2-asp-core-mvc/schedule/EJ2_ASP.MVC/header-bar.md [hunk 1]
```diff
@@ -162,6 +162,33 @@ In month view, the date header template is not applicable and therefore the same
 {% endtabs %}
 {% endif %}
 
+## Customizing the date range text
 
+The `dateRangeTemplate` option allows you to customize the text content of the date range displayed in the scheduler. By default, the date range text is determined by the scheduler view being used. However, you can use the `dateRangeTemplate` option to override the default text and specify your own custom text to be displayed.
```
Comment (VengateshManiraj) [inline ej2-asp-core-mvc/schedule/EJ2_ASP.MVC/header-bar.md:167]: `DateRangeTemplate`

File: ej2-asp-core-mvc/schedule/EJ2_ASP.MVC/resources.md [hunk 2]
```diff
@@ -615,6 +615,42 @@ Working hours indicates the work hour duration of a day, which is highlighted vi
 
 In this example, a resource named `Will Smith` is depicted with working hours ranging from 8.00 AM to 3.00 PM and is visually illustrated with active colors, whereas the other two resources have different working hours set.
 
+## Hide non-working days when grouped by date
+
+In Scheduler, you can set custom work days for each resource and group the Scheduler by date to display these work days. By default, the Scheduler will show all days when it is grouped by date, even if they are not included in the custom work days for the resources. However, you can use the `hideNonWorkingDays` property to only display the custom work days in the Scheduler.
```
Comment (VengateshManiraj) [inline ej2-asp-core-mvc/schedule/EJ2_ASP.MVC/resources.md:620]: `HideNonWorkingDays`

File: ej2-asp-core-mvc/code-snippet/tab/tabkeynavigation/razor [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        new TabTabItem { Header = ViewBag.headerText2, Content = "Java is a set of computer software and specifications developed by Sun Microsystems, later acquired by Oracle Corporation, that provides a system for developing application software and deploying it in a cross-platform computing environment. Java is used in a wide variety of computing platforms from embedded devices and mobile phones to enterprise servers and supercomputers. While less common, Java applets run in secure, sandboxed environments to provide many features of native applications and can be embedded in HTML pages.", tabindex = 0 },
+        new TabTabItem { Header = ViewBag.headerText3, Content = "The command-line compiler, VBC.EXE, is installed as part of the freeware .NET Framework SDK. Mono also includes a command-line VB.NET compiler. The most recent version is VB 2012, which was released on August 15, 2012.", tabindex = 0 },
```
Comment (VengateshManiraj) [inline ej2-asp-core-mvc/code-snippet/tab/tabkeynavigation/razor:9]: Check the property name `TabIndex` or `tabIndex`.

File: ej2-asp-core-mvc/tab/how-to/tab-key-navigation.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+platform: ej2-asp-core-mvc
+control: Tab key navigation
+publishingplatform: ##Platform_Name##
+documentation: ug
+---
+
+
+# Enabling tab key navigation in Tabs
+
+The `tabIndex` property of a Tab item is used to enable tab key navigation for that particular item. When a positive value is assigned to the `tabIndex` property, it allows the user to switch focus to the next or previous tab item using the Tab or Shift+Tab keys. By default, the user can only switch between tab items using the arrow keys.
```
Comment (VengateshManiraj) [inline ej2-asp-core-mvc/tab/how-to/tab-key-navigation.md:14]: `TabIndex` or `tabIndex`

File: ej2-asp-core-mvc/toolbar/item-configuration.md [hunk 5]
```diff
@@ -105,6 +105,71 @@ Output be like the below.
 
 ![Toolbar Control with item configuration](./images/toolbar_tempalte.PNG)
 
+### Enabling tab key navigation in Toolbar
+
+The `tabIndex` property of a Toolbar item is used to enable tab key navigation for the item. By default, the user can switch between items using the arrow keys, but the `tabIndex` property allows you to switch between items using the Tab and Shift+Tab keys as well.
```
Comment (VengateshManiraj) [inline ej2-asp-core-mvc/toolbar/item-configuration.md:110]: `TabIndex` or `tabIndex`

Comment (VengateshManiraj) [reply ej2-asp-core-mvc/toolbar/item-configuration.md:110]: Ensure this same in sample code too

**Review by VengateshManiraj — CHANGES_REQUESTED** [review]

**Comment by VenkateshwaranSaravanakumar** [comment]
Changes moved in the hotfix
#1407

**Pull request closed without merging**